android-radar/src/main/java/com/cedexis/androidradar/CedexisRadarWebClient.java

```diff
@@ -29,7 +30,7 @@
     // 2 is the client profile code for AndroidRadar
     final int CLIENT_PROFILE = 2;
     // Client profile version conveys the version of the WebView wrapper code.
-    final int CLIENT_PROFILE_VERSION = 1;
+    final int CLIENT_PROFILE_VERSION = 2;
```

Comment:
Oh is this how this was meant to be used? I thought it was the version of the `CLIENT_PROFILE` flag, I had taken this to be the version of the set of `CLIENT_PROFILE` in case we renumber things or something. This makes more sense though.

Reply:
Yeah, this is currently the only way we have to indicate what version *of the AndroidRadar code* is responsible for the data sent.